src/main/java/uk/gov/hmcts/darts/arm/client/ArmApiClient.java

```diff
@@ -21,7 +22,12 @@ public interface ArmApiClient {
         consumes = APPLICATION_JSON_VALUE,
         produces = APPLICATION_JSON_VALUE
     )
-    ResponseEntity<UpdateMetadataResponse> updateMetadata(@RequestHeader(AUTHORIZATION) String bearerAuth,
+    UpdateMetadataResponse updateMetadata(@RequestHeader(AUTHORIZATION) String bearerAuth,
```

Comment:
Not a comment for you to address here (just my opinion :)) but we could have created an arm open api spec and had much of this stuff generated for us (like we do in the gateway)

Reply:
I'm not sure what direction this may take us - I'd just be careful wasting any effort doing that because I'm pretty sure the properties for the `updateMetadata` request are inconsistent with a mix of styles, so it is helpful having our custom models. Unless this is talking about some feign client stuff, so pass!

Reply:
In my mind most of this stuff could be easily generated from a custom openapi spec. The spec acts as a good easy to understood service descriptor that can conveniently generate both server and client feign side code. Plus acts as a good discussion point with external parties

Appreciate the bespoke serialization/derisalization of this data may not totally conform to the arm spec and you may have to write custom code for this but the rest of the code should be autogenerated for you

Anyway like I say, just an idea